ckanext/harvest/templates/source/new_source_form.html

```diff
@@ -14,6 +14,8 @@
       </span>
   {% endcall %}
 
+  {{ h.csrf_input() }}
```

Comment:
Thansk @pdekraker-epa! 

This will fail in older CKAN versions that don't have this helper available, so we need:

```suggestion
{{ h.csrf_input() if 'csrf_input' in h }}
```

Also can you merge the latest master branch? That should fix the tests